conversation.go

```diff
@@ -5,6 +5,8 @@ import (
 	"net/url"
 	"strconv"
 	"strings"
+
+	"github.com/pkg/errors"
```

Comment:
We don't use pkg/errors now.
In this case, you can use `fmt.Errorf` or standard `errors` package instead.

Reply:
Changed it to the built in `errors` package